charts/ark-cloudconfig/values.yaml

```diff
@@ -0,0 +1,74 @@
+# Default values for ark-cloudconfig.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
```

Comment:
Can CloudConfig support multiple replicas? If not, please add a comment warning the reader not to set the `replicaCount` to anything but `1`, or maybe even better would be to just remove this option.

Reply:
okay.